### PR TITLE
Minh/like

### PR DESCRIPTION
**Pull request by minhphu20**
Adding logic for like and not like images.

### CODE REVIEW

File: src/main/java/com/google/codeu/data/User.java [hunk 1]
```diff
@@ -17,4 +26,31 @@ public String getEmail() {
   public String getAboutMe() {
     return aboutMe;
   }
+
+  public HashSet<String> getLikes() { return likes; }
```
Comment (cindyxue) [inline src/main/java/com/google/codeu/data/User.java:30]: Separate it to three lines to match styles with the other function

File: src/main/java/com/google/codeu/data/User.java [hunk 2]
```diff
@@ -17,4 +26,31 @@ public String getEmail() {
...
+
+  public HashSet<String> getNotLikes() { return notLikes; }
```
Comment (cindyxue) [inline src/main/java/com/google/codeu/data/User.java:32]: Separate it to three lines to match styles with the other function

File: src/main/java/com/google/codeu/data/User.java [hunk 3]
```diff
@@ -17,4 +26,31 @@ public String getEmail() {
...
+
+  public void setAboutMe(String about) {
+    this.aboutMe = about;
+  }
+
+  public void addLike(String user) {
```
Comment (cindyxue) [inline src/main/java/com/google/codeu/data/User.java:38]: Make parameter name more specifically (eg. userEmail).

File: src/main/java/com/google/codeu/data/User.java [hunk 4]
```diff
@@ -17,4 +26,31 @@ public String getEmail() {
...
+    System.out.println("Inside adding likes");
+    if (this.likes == null) {
+      this.likes = new HashSet<String>();
+    }
+    this.likes.add(user);
+    System.out.println("Finish inside adding likes");
+  }
+
+  public void addNotLike(String user) {
```
Comment (cindyxue) [inline src/main/java/com/google/codeu/data/User.java:47]: Make parameter name more specifically (eg. userEmail).

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 5]
```diff
@@ -225,4 +199,82 @@ public void storeMarker(UserMarker marker) {
...
+  public User getUser(String email) {
+    Query query = new Query("User")
+            .setFilter(new Query.FilterPredicate("email", FilterOperator.EQUAL, email));
+    PreparedQuery results = datastore.prepare(query);
+    Entity userEntity = results.asSingleEntity();
+    if(userEntity == null) {
+      return null;
+    }
+    String aboutMe = (String) userEntity.getProperty("aboutMe");
+    HashSet<String> likes = new HashSet<String>();
```
Comment (cindyxue) [inline src/main/java/com/google/codeu/data/Datastore.java]: Is it better to initialize likes and dislikes hashset with null? Thus, we could distinguish if the user has likes or dislikes easily.

**Review by cindyxue — CHANGES_REQUESTED** [review]

**Comment by minhphu20** [comment]
Archived as major code refractoring has been done in another pull :)

**Pull request closed without merging**